rune/Rune.ts

```diff
@@ -96,6 +96,21 @@ export class Rune<out T, out U = never> {
     return value instanceof Rune ? value.into(ValueRune) : Rune.constant(value)
   }
 
+  static str<X, T extends TemplateStringsArray, R extends unknown[]>(
+    ...args: RunicArgs<X, [T, ...R]>
+  ) {
```

Comment:
No need to take `T` and `R` as generics (this isn't zones :P) and no need to have `TempalteStringsArray` in the runic args
```suggestion
  static str<X>(strings: TemplateStringsArray, ...args: RunicArgs<X, unknown[]>) {
```

Reply:
Without extracting `T` and `R` into type params/runic args, the `U` doesn't propagate into the `Rune.str`-resulting Rune.

Reply:
Ah, because it was an `unknown[]`, an explicit `RunicArgs.resolve` was needed. I've pushed this fix.